lib/cowsay.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function cowsay() {
+
+  const cow = '$the_cow = <<"EOC"'; +
```

Comment:
the semicolon means end of expression so if you want the string to continue past the line maybe remove it